components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/validators/TokenValidationHandler.java

```diff
@@ -59,6 +59,7 @@
     private Log log = LogFactory.getLog(TokenValidationHandler.class);
     private Map<String, OAuth2TokenValidator> tokenValidators = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
     private static final String BUILD_FQU_FROM_SP_CONFIG = "OAuth.BuildSubjectIdentifierFromSPConfig";
+    private static final String AUTH_FLOW = "OAuth.AuthFlow";
```

Comment:
Can we move this constant to a common location? I see it is repeated in https://github.com/wso2-extensions/identity-carbon-auth-rest/pull/69 as well.

Reply:
Move the constant to OauthConstants and with this modification Oauth version in wso2-extensions/identity-carbon-auth-rest need to be updated to the immediate version which includes this modification